packages/recomponents/src/components/r-toast/r-toast.spec.js

```diff
@@ -55,4 +55,103 @@ describe('r-toast.vue', () => {
 
         expect(managerToasts).toHaveLength(4);
     });
+
+    it('should create toasts from errors via toast-manager', async () => {
+        const error = new Error();
+        const toastMessage = 'Custom error text';
+        localVue.$toast.fromError(error, toastMessage);
```

Comment:
Just one minor suggestion: I would extract `localVue.$toast.fromError` into a function to hide the implementation details and making the test code a little simpler to read.